artview/components/menu.py

```diff
@@ -264,13 +268,13 @@ def _about(self):
         txOut = "This is a simple radar file browser to allow \
                  quicklooks using the DoE PyArt software"
         QtGui.QMessageBox.about(self, "About ARTView", txOut)
- 
+
     def _get_RadarLongInfo(self):
-        '''Print out the radar info to text box.'''
+        '''Print out the radar info to text box and terminal.'''
```

Comment:
actually we already have a way of displaying long texts [here](https://github.com/nguy/artview/blob/master/artview/core/common.py#L53), I'm using this for the help button in the plug-ins.
The problem is that txOut is None, `.info()` is doing the print, not `print txOut`, see the [pyart documentation](http://arm-doe.github.io/pyart-docs-travis/user_reference/generated/pyart.core.Radar.info.html)
this can only be solved with a [string buffer](https://docs.python.org/2/library/stringio.html)